### DISCUSSION TITLE
Cannot send a notification message

### DISCUSSION THREAD

**Discussion by santipongth** [kreait/firebase-php#892] [General]
Please help me to check my PHP source code. I cannot send notification messages to a single, specific device, pass the device's registration token. I do not any error but the message cannot send out to a mobile phone. Here is the source code.

```
<?php

require __DIR__.'/vendor/autoload.php';

use Kreait\Firebase\Factory;
use Kreait\Firebase\Messaging\CloudMessage;
use Kreait\Firebase\Messaging\Notification;

$factory = (new Factory)->withServiceAccount('adminsdk.json');

$messaging = $factory->createMessaging();

$deviceToken = 'xxxxxx';

$message = CloudMessage::withTarget('token', $deviceToken)
    ->withNotification(Notification::create('Title', 'Body'))
    ->withData(['key' => 'value']);

$result = $messaging->send($message);
?>
```
Thank you for your help.

**Comment by jeromegamez** [comment]
Unfortunately, if the message is accepted by the Firebase backend/doesn't throw an error, it means that the registration token is valid and registered with your Firebase project.

From the SDK's perspective, everything is working as expected, so it seems likely that this is an issue that needs to be solved on the receiving client.

**Reply by santipongth** [reply]
Thank you
